clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/cache/client/TaskCacheClient.java

```diff
@@ -51,6 +51,7 @@ protected Task convert(CacheData cacheData) {
     task.setGroup((String) attributes.get("group"));
     task.setLastStatus((String) attributes.get("lastStatus"));
     task.setDesiredStatus((String) attributes.get("desiredStatus"));
+    task.setHealthStatus((String) attributes.get("healthStatus"));
```

Comment:
Can we add a test to [TaskCacheClientTest.java](https://github.com/spinnaker/clouddriver/blob/master/clouddriver-ecs/src/test/java/com/netflix/spinnaker/clouddriver/ecs/cache/TaskCacheClientTest.java) that asserts this is being set as expected?

Reply:
Addressed